source/agora/test/QuorumShuffle.d

```diff
@@ -0,0 +1,469 @@
+/*******************************************************************************
+
+    Tests regular quorum shuffling behavior.
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module agora.test.QuorumShuffle;
+
+version (unittest):
+
+import agora.api.Validator;
+import agora.common.Amount;
+import agora.common.Config;
+import agora.consensus.data.ConsensusParams;
+import agora.common.crypto.Key;
+import agora.common.Hash;
+import agora.consensus.data.Block;
+import agora.consensus.data.Enrollment;
+import agora.consensus.data.Transaction;
+import agora.consensus.data.genesis.Test;
+import agora.node.FullNode;
+import agora.test.Base;
+
+import std.algorithm;
+import std.format;
+import std.range;
+
+import core.thread;
+import core.time;
```

Comment:
Some of those imports should be useless

Reply:
Oh yes. Only 2 of them are actually used. Thanks.